internal/api/approvals.go

```diff
@@ -354,9 +354,9 @@ func ApprovalHandler(
 
 			// Authorize request.
 			userEmail := r.Context().Value("userEmail").(string)
-			if doc.Status != "In-Review" && doc.Status != "In Review" {
```

Comment:
We never actually needed to confirm the "In Review" status because all docs created through Hermes would only have an "In-Review" (with the hyphen) status.